tests/sentry/api/endpoints/test_organization_releases.py

```diff
@@ -167,6 +168,7 @@ def test_release_list_order_by_sessions_empty(self):
             response, [release_5, release_4, release_3, release_2, release_1]
         )
 
+    @pytest.mark.xfail(reason="Does not work with the metrics release health backend")
```

Comment:
@wedamija fyi - is this actually expected to work with the release health backend? if not, i can remove it

Reply:
Do we know why it doesn't work? I think it's probably testing something useful here that should work across backends

Reply:
Likely because store_session is writing to snuba's sessions backend. I'm not really sure how to change it though, and would need someone more familiar with that code to take it up.

Reply:
same for the sessions api tests fyi

Reply:
Yeah it's definitely not good, the team that did the conversion should have at least had tests running for both backends. I'm not sure who even owns this now, but I really don't think we should be disabling tests here

Reply:
@wedamija how do we get this fixed then?
There are serious risks associated with having the default backend be something that just doesn't run in production. We have to remember to override it in ops, self hosted and everywhere we deploy sentry or stuff will be broken. In the meantime, are we just pretending we have existing coverage and are "disabling" something here when we don't have any real coverage anyway?

Reply:
I'm going to see if i can switch out to the old backend and preserve the tests at least

Reply:
Ok that doesn't work. @iambriccardo do you have any idea how to do what dan suggests about migrating `store_session`?

Reply:
Hi! Let me take a look. A teammate of mine did the conversion, I know partially about the whole domain but let me see if I can be of any help.

Reply:
@lynnagara I fixed the problem and pushed the changes directly here. I hope that is fine by you.